README.md

```diff
@@ -644,22 +644,22 @@ assert_eq!(combine_all(&product_nums), Product(24))
 
 ### Features
 
-Frunk comes with support for deriving [serde](https://github.com/serde-rs/serde) serializer/deserializers for its core 
+Frunk comes with support for deriving [serde](https://github.com/serde-rs/serde) serializer/deserializers for its core
 data structures. This can be enabled by adding the `serde` feature flag.
 
-For example, if you'd like to use just `frunk-core` with serde
+For example, if you'd like to use just `frunk_core` with serde
 
 ```toml
 [dependencies]
-frunk-core = { version = "$version", features = ["serde"] }
+frunk_core = { version = "$version", features = ["serde"] }
 ```
 
-Or, if you'd like to use `frunk` with serde, you need to explicitly include `frunk-core` as well
+Or, if you'd like to use `frunk` with serde, you need to explicitly include `frunk_core` as well
```

Comment:
🤦🏻‍♂️doh ! thanks !

Reply:
Well, thank you for this wonderful crate! 😄 🙇‍♂️